docroot/themes/humsci/humsci_basic/src/js/media/video-with-caption.js

```diff
@@ -7,7 +7,7 @@ const videos = document.querySelectorAll('.field-media-oembed-video');
 if (videos && videos.length > 0) {
   for (let i = 0; i < videos.length; i++) {
     const video = videos[i];
-    if (video.parentNode && video.parentNode.parentNode && video.parentNode.parentNode.classList.nodeName == 'figure')
+    if (video.parentNode && video.parentNode.parentNode && video.parentNode.parentNode.classList.nodeName == 'figure') {
```

Comment:
hmm i missed this yesterday too but this actually doesn't work, the last condition statement should be:

`video.parentNode.parentNode.nodeName == 'FIGURE'`